tokens/tokens-eth.json

```diff
@@ -8351,40 +8351,7 @@
   "twitter"   : "",
   "youtube"   : ""
 }
-},
-    {
-    "symbol"      : "NGC",
-    "address"     : "0x72dd4b6bd852a3aa172be4d6c5a6dbec588cf131",
-    "decimals"    : "18",
-    "name"        : "NAGA Coin",
-    "ens_address" : "",
-    "website"     : "https://www.nagaico.com",
-    "logo": {
-      "src"       : "",
-      "width"     : "",
-      "height"    : "",
-      "ipfs_hash" : ""
-    },
-    "support": {
-      "email"     : "support@nagaico.com",
-      "url"       : ""
-    },
-    "social": {
-      "blog"      : "",
-      "chat"      : "",
-      "facebook"  : "https://www.facebook.com/thenagaico/",
-      "forum"     : "",
-      "github"    : "",
-      "gitter"    : "",
-      "instagram" : "",
-      "linkedin"  : "",
-      "reddit"    : "",
-      "slack"     : "",
-      "telegram"  : "",
-      "twitter"   : "https://twitter.com/thenagaico",
-      "youtube"   : ""
-    }
-  },{
+},{
```

Comment:
Please revert removal of the NGC, NAGA Coin entry
`    {
    "symbol"      : "NGC",
    "address"     : "0x72dd4b6bd852a3aa172be4d6c5a6dbec588cf131",
    "decimals"    : "18",
    "name"        : "NAGA Coin",
    "ens_address" : "",
    "website"     : "https://www.nagaico.com",
    "logo": {
      "src"       : "",
      "width"     : "",
      "height"    : "",
      "ipfs_hash" : ""
    },
    "support": {
      "email"     : "support@nagaico.com",
      "url"       : ""
    },
    "social": {
      "blog"      : "",
      "chat"      : "",
      "facebook"  : "https://www.facebook.com/thenagaico/",
      "forum"     : "",
      "github"    : "",
      "gitter"    : "",
      "instagram" : "",
      "linkedin"  : "",
      "reddit"    : "",
      "slack"     : "",
      "telegram"  : "",
      "twitter"   : "https://twitter.com/thenagaico",
      "youtube"   : ""
    }
  }`